interface.dd

```diff
@@ -70,6 +70,17 @@ interface D
 }
 ------
 
+    $(P Inferfaces can have function templates in the members.
+        The all instantiated functions will be $(D final) implicitly.
```

Comment:
s/The all instantiated/All instantiated/

s/will be $(D final) implicitly/are implicitly $(D final)/